readme.md

```diff
@@ -207,6 +207,7 @@
 - [Sass](https://github.com/Famolus/awesome-sass) - CSS preprocessor.
 - [Ant Design](https://github.com/websemantics/awesome-ant-design) - Enterprise-class UI design language.
 - [Less](https://github.com/LucasBassetti/awesome-less) - CSS preprocessor.
+- [Progressive Enhancement](https://github.com/jbmoelker/progressive-enhancement-resources) - From concept and strategies to feature detection & testing methods. Complete with a list of (code) examples.
```

Comment:
Fair enough. `Progressive Enhancement - Concept, strategies, feature detection and (code) examples.` better? That's a description of 59 characters. Longest in the list is currently just over 100 characters. I don't know what's acceptable.